packages/console/src/cloud/pages/Main/TenantLandingPage/TenantLandingPageContent/index.tsx

```diff
@@ -22,11 +23,18 @@ type Props = {
 
 function TenantLandingPageContent({ className }: Props) {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
-  const { tenants, mutate } = useTenants();
+  const { data: availableTenants, mutate } = useCloudSwr('/api/tenants');
+  const { tenants, setTenants, navigate } = useContext(TenantsContext);
   const theme = useTheme();
 
   const [isCreateModalOpen, setIsCreateModalOpen] = useState(false);
 
+  useEffect(() => {
+    if (availableTenants) {
+      setTenants(availableTenants);
+    }
+  }, [availableTenants, setTenants]);
```

Comment:
nit: Since we have `useCloudSwr`, can we consider removing the `tenants` from context?